<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComDialog</class>
 <widget class="QDialog" name="ComDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure connect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="selectBox">
       <property name="title">
        <string>Выбор COM-порта:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QLabel" name="serialNumberLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="manufacturerLabel">
          <property name="text">
           <string>Производитель:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="locationLabel">
          <property name="text">
           <string>Location:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="vendorLabel">
          <property name="text">
           <string>Vendor ID:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QComboBox" name="comPortSelectBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string>Описание:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="productLabel">
          <property name="text">
           <string>Product ID:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="parametersBox">
       <property name="title">
        <string>Настройка COM-порта:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QComboBox" name="baudRateBox"/>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="stopBitsBox"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="baudRateLabel">
          <property name="text">
           <string>BaudRate:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="dataBitsBox"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="parityBox"/>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="flowControlBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="dataBitsLabel">
          <property name="text">
           <string>Data bits:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="parityLabel">
          <property name="text">
           <string>Parity :</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="stopBitsLabel">
          <property name="text">
           <string>Stop bits</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="flowControlLabel">
          <property name="text">
           <string>Flow control:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
